ipfsApi/utils.py

```diff
@@ -3,143 +3,12 @@
 
 from __future__ import absolute_import
 
-import io
-import json
 import mimetypes
 import os
 from functools import wraps
 
 import six
 
-from six.moves import cPickle as pickle
```

Comment:
I agree with getting rid of all of these wrapper functions; they were a little silly. :smile: 

On a more project-management note, I think that in the future we should try to do this kind of refactor in its own PR. Thoughts?